django/db/models/fields/related.py

```diff
@@ -1586,7 +1587,20 @@ class ForeignObject(RelatedField):
 
     def __init__(self, to, from_fields, to_fields, rel=None, related_name=None,
             related_query_name=None, limit_choices_to=None, parent_link=False,
-            on_delete=CASCADE, swappable=True, **kwargs):
+            on_delete=None, swappable=True, **kwargs):
```

Comment:
Is there a reason you didn't move `on_delete` so it can be passed positionally as done elsewhere?